liquid-staking/src/basics/constants.rs

```diff
@@ -0,0 +1,8 @@
+pub const DEFAULT_GAS_TO_CLAIM_REWARDS: u64 = 6_000_000;
+pub const MIN_GAS_FOR_ASYNC_CALL: u64 = 12_000_000;
+pub const MIN_GAS_FOR_CALLBACK: u64 = 12_000_000;
+pub const MIN_EGLD_TO_DELEGATE: u64 = 1_000_000_000_000_000_000;
```

Comment:
Add comment so we don't have to count zeroes.
```suggestion
pub const MIN_EGLD_TO_DELEGATE: u64 = 1_000_000_000_000_000_000; // 1 EGLD
```